viadot/tasks/sqlite_tasks.py

```diff
@@ -0,0 +1,51 @@
+import prefect
+from prefect import Task
+
+from ..sources.sqlite import SQLite
+import pandas as pd
+import sqlite3
+
+from sqlalchemy import create_engine
+from typing import Any, Dict
+import os
+
+
+class LoadDF(Task):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(name="load_DF", *args, **kwargs)
+
+    def run(self,
+            table_name: str = None,
+            schema: str = None,
+            dtypes: Dict[str, Any] = None,
+            db_path: str = None,
+            df: pd.DataFrame = None,
+            if_exists: str = "fail"):
+
+        sqlite = SQLite(driver="{SQLite}", server="localhost", db=db_path)
+        sqlite.create_table(table=table_name, schema=schema, dtypes=dtypes, if_exists=if_exists)
+        sqlite.insert_into(table=table_name, df=df)
+
+        return True
+
+class RunSQL(Task):
```

Comment:
can you check this task with @Trymzet

I think the code makes sense but since now we are returning a DataFrame we might need to rename it. Something like `SQLtoDF()`. @Trymzet please discuss with @AdamSulek 

Also you need to add a validation rule that the SQL starts with `select` because if it starts with `create` or anything else your sql execution would return no results.

Reply:
Yes, the name doesn't make sense as it seems you're only fetching SELECT results, not running an arbitrary query. If you're running other types of operations you also need to ensure you're committing the transaction. You can check out the query task I added for Azure SQL database on 0.12 branch.